mmv1/third_party/terraform/services/networkservices/resource_network_services_lb_route_extension_test.go

```diff
@@ -0,0 +1,847 @@
+
+      cat <<EOF > /var/www/html/index.html
+      <pre>
+      Name: $NAME
+      IP: $IP
+      Metadata: $METADATA
+      </pre>
+      EOF
+    EOF1
+  }
+
+  lifecycle {
+    create_before_destroy = true
+  }
+}
+
+# health check
+resource "google_compute_region_health_check" "default" {
+  name     = "tf-test-l7-ilb-hc%{random_suffix}"
+  region   = "us-west1"
+
+  http_health_check {
+   port_specification = "USE_SERVING_PORT"
+  }
+}
+
+# MIG
+resource "google_compute_region_instance_group_manager" "mig" {
+  name     = "tf-test-l7-ilb-mig1%{random_suffix}"
+  region   = "us-west1"
+
+  base_instance_name = "vm"
+  target_size        = 2
+
+  version {
+    instance_template = google_compute_instance_template.instance_template.id
+    name              = "primary"
+  }
+}
+
+# allow all access from IAP and health check ranges
+resource "google_compute_firewall" "fw_iap" {
+  name          = "tf-test-l7-ilb-fw-allow-iap-hc%{random_suffix}"
+  direction     = "INGRESS"
+  network       = google_compute_network.ilb_network.id
+  source_ranges = ["130.211.0.0/22", "35.191.0.0/16", "35.235.240.0/20"]
+
+  allow {
+    protocol = "tcp"
+  }
+}
+
+# allow http from proxy subnet to backends
+resource "google_compute_firewall" "fw_ilb_to_backends" {
+  name          = "tf-test-l7-ilb-fw-allow-ilb-to-backends%{random_suffix}"
+  direction     = "INGRESS"
+  network       = google_compute_network.ilb_network.id
+  source_ranges = ["10.0.0.0/24"]
+  target_tags   = ["http-server"]
+
+  allow {
+    protocol = "tcp"
+    ports    = ["80", "443", "8080"]
+  }
+
+  depends_on = [
+    google_compute_firewall.fw_iap
+  ]
+}
+
+resource "google_network_services_lb_route_extension" "default" {
+  name                  = "tf-test-l7-ilb-route-ext%{random_suffix}"
+  description           = "my route extension"
+  location              = "us-west1"
+  load_balancing_scheme = "INTERNAL_MANAGED"
+  forwarding_rules      = [google_compute_forwarding_rule.default.self_link]
+
+  extension_chains {
+    name = "chain1"
+
+    match_condition {
+      cel_expression = "request.path.startsWith('/extensions')"
+    }
+
+    extensions {
+      name      = "ext12"
+      authority = "ext12.com"
+      service   = google_compute_region_backend_service.callouts_backend_2.self_link
+      timeout   = "0.2s"
+      fail_open = false
+
+      forward_headers  = ["custom-header"]
+    }
+  }
+
+  extension_chains {
+    name = "chain2"
+
+    match_condition {
+      cel_expression = "request.path.startsWith('/extensions2')"
+    }
+
+    extensions {
+      name      = "ext11"
+      authority = "ext11.com"
+      service   = google_compute_region_backend_service.callouts_backend.self_link
+      timeout   = "0.1s"
+      fail_open = false
+
+      forward_headers  = ["custom-header"]
+    }
+  }
+
+  labels = {
+    foo = "bar"
```

Comment:
Can you please update the fields `labels` with a different value?

Reply:
updated